code/modules/cooking_with_jane/cooking_appliances/appliance_defines.dm

```diff
@@ -0,0 +1,11 @@
+/obj/machinery/cooking_with_jane
+	name = "Default Cooking Appliance"
+	desc = "Lookit me, I'm a cool machinery-doo. Hex didn't take his focus pills today."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "processor"
+	density = TRUE
+	anchored = TRUE
+	layer = BELOW_OBJ_LAYER
+	var/cooking = FALSE
+
+	//circuit = /obj/item/electronics/circuitboard/cooking_with_jane/
```

Comment:
Needed?

Reply:
You mean the commented circuit?

Reply:
Yes. If you plan to use it later it's fine to let it there commented.

Reply:
Removed.